addon/components/paper-switch.js

```diff
@@ -69,11 +77,11 @@ export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
   },
 
   _drag(event) {
-    if (this.get('disabled')) { return; }
+    if (this.getAttr('disabled')) { return; }
```

Comment:
Let's use normal `get` for now, to conform with the rest of the project and in the light of this article: http://locks.svbtle.com/to-attrs-or-not-to-attrs

Please point out your reasons on the main issue, if you find appropriate.